app/src/main/java/org/simple/clinic/shortcodesearchresult/ShortCodeSearchResultScreen.kt

```diff
@@ -234,19 +229,8 @@ class ShortCodeSearchResultScreen(
     // screen to Mobius.
     val prefix = shortCode.substring(0, 3)
     val suffix = shortCode.substring(3)
-    val formattedShortCode = "$prefix${Unicode.nonBreakingSpace}$suffix"
 
-    return Truss()
-        .pushSpan(textSpan)
-        .append(formattedShortCode)
-        .popSpan()
-        .build()
-
-  }
-
-  private fun formatShortCodeForDisplay(context: Context, shortCode: String): CharSequence {
-    val textSpacingSpan = TextAppearanceWithLetterSpacingSpan(context, R.style.TextAppearance_Simple_Body0_Numeric)
```

Comment:
The reason we were doing this in this way was because the short code had a different letter spacing than the rest of the text.

Have we changed the design to no longer require a different letter spacing for the short code?

Reply:
The design is the same, instead of applying the text appearance at runtime, I have applied it to the `Toolbar` directly. `app:titleTextAppearance="?attr/textAppearanceBody0Numeric"` Since we don't have 2 different styles for the title it seemed easier to apply it directly rather than using spans.

Reply:
Right, I think originally the title had two different text appearances which was why we went with this approach.